README.md

```diff
@@ -52,6 +57,23 @@ For information about the API, see the [API documentation](API.md).
 
 For information about testing the service, see the [Testing documentation](TESTING.md).
 
+## Secrets
+
+Secrets for this project follow the naming convention `${EnvironmentType}/${DNSRecord}/honeybadger_api_key`. Our security model controls access to secrets based on the prefix, and will deny read access to any secret not prefixed with `development/`, so we tell our application's Cloudformation template whether we are in a development, test, or production environment type via [config files](cicd/3-app/aiproxy/config).
+
+* In production: `production/aiproxy.code.org/honeybadger_api_key`
+* In test: `test/aiproxy-test.code.org/honeybadger_api_key`
+* In an "adhoc" development environment: `development/aiproxy-dev-mybranch.code.org/honeybadger_api_key`
+* A prod-like environment, not based on `main`: `production/aiproxy-otherbranch.code.org/honeybadger_api_key`
+
+Read more about environments and environment types in [cicd/README.md](cicd/README.md).
+
+### Creating new secrets
+
+To create a new secret, define it in "[cicd/3-app/aiproxy/template.yml](cicd/3-app/aiproxy/template.yml) in the "Secrets" section. This will create an empty secret once deployed.
+
+Once created, you can set the value of this secret (via the AWS Console, as an Admin). Finally, deploy your code that uses the new secret, loading it via the `GetSecretValue` function from the AWS SDK.
```

Comment:
This admittedly isn't the best Dev experience, we could implement a similar script to bin/update_secrets over in the code-dot-org repo if we want to make this easier. Keeping it simple for MVP.